src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
+                 $HOME and $SHELL are set to the target login. $USER is \
+                 set to the target login. $TERM is imported from the \
+                 current environment. The environment variables from the \
+                 login class capability database for the target login \
+                 are also set.'),
+    },
+    {
       type: 'input',
       name: 'exec_timeout',
-      placeholder: T('exec_timeout'),
-      tooltip: T('Define the maximum amount of time in seconds to wait for\
- a command to complete. If a command is still running after the allotted\
- time, the jail will be terminated.'),
-    }, {
+      placeholder: T('exec.timeout'),
+      tooltip: T('Maximum amount of time in seconds to wait for a \
+                  command to complete. If a command is still running \
+                  after the allotted time, the jail will not be \
+                  created or removed.'),
+    },
+    {
       type: 'input',
       name: 'stop_timeout',
-      placeholder: T('stop_timeout'),
-      tooltip: T('Define the maximum amount of time in seconds to wait for\
- the jail processes to exit after sending a SIGTERM signal. This happens\
- after any <b>exec_stop</b> commands are complete. After the defined time,\
- the jail is removed, killing any remaining processes. If this is set to\
- <i>0</i>, no SIGTERM is sent and the jail is immediately removed.'),
-    }, {
+      placeholder: T('stop.timeout'),
+      tooltip: T('Maximum amount of time in seconds to wait for jail \
+                 processes to exit after sending a SIGTERM signal. The \
+                 timeout starts after <b>exec.stop</b> commands have \
+                 completed. When time is up the jail is removed, killing \
+                 any remaining processes. When set to <i>0</i>, no \
+                 SIGTERM is sent and the jail is immediatley removed. \
+                 The default timeout is 10 seconds.'),
+    },
+    {
       type: 'input',
       name: 'exec_jail_user',
-      placeholder: T('exec_jail_user'),
-      tooltip: T('Enter either <i>root</i> or a valid <i>username</i>. In\
- the jail environment, commands run as this defined user.'),
-    }, {
+      placeholder: T('exec.jail_user'),
+      tooltip: T('The user to run commands as in the jail environment. \
+                 The default is to run commands as the current user. \
+                 Set to a valid username to run commands as that user.'),
+    },
+    {
       type: 'input',
       name: 'exec_system_jail_user',
-      placeholder: T('exec_system_jail_user'),
-      tooltip: T('This boolean option looks for the <b>exec_jail_user</b>\
- in the system <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=passwd&sektion=5&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">passwd(5)</a> file instead of the file from the jail.'),
-    }, {
+      placeholder: T('exec.system_jail_user'),
+      tooltip: T('Set this boolean option to True to look for the \
```

Comment:
We're generally referring to input values for fields with italics: <i>True</i>

Reply:
fixed